doc/migrations/4.2.0_4.3.0.md

```diff
@@ -133,6 +133,39 @@ GO
 ```
 
 
+### Private storage indexes
```

Comment:
Not sure, but looks like missed info go `MSSQL`.

Reply:
ok, added.
I needed a working shell for mssql first